requirements.dev.in

```diff
@@ -24,8 +24,8 @@ pytest-responses
 https://github.com/opensafely-core/pipeline/archive/refs/tags/v0.0.3.zip#egg=opensafely-pipeline
 
 # report code tests
-numpy
-pandas
+numpy==1.22.4
```

Comment:
Numpy and pandas are used here to test the analytics code we run in production. Therefore it would make sense to use the same version as is used in production.

I'm not sure what versions those are, since it just seems to use the python:latest image? Is that from the python-docker repo? i.e https://github.com/opensafely-core/python-docker/blob/main/requirements.txt 
If so, those versions are quite a bit older.

Reply:
Aha, I definitely misunderstood the point of this change! Will update with the correct pins.

Reply:
See comment https://github.com/opensafely-core/interactive.opensafely.org/issues/309#issuecomment-1192583814